src/storage/addtoCard.js

```diff
@@ -33,6 +33,7 @@ const AddtoCardCtxProvider = ({ children }) => {
   // states
   const [productCount, setProductCount] = useState(0);
   const [addedProduct, setAddedProduct] = useState(getLocalStorge());
```

Comment:
addedProducts not addedProduct,
setAddedProducts not setAddedProduct.

also can be `cardProducts` and `setCardProducts`